docs/book/beyond_basics/upgrading_kubebuilder.md

```diff
@@ -17,9 +13,11 @@ Update your project's dependencies to the latest version of the libraries used b
 will modify *Gopkg.toml* by rewriting the `[[override]]` elements beneath the
 `# DO NOT MODIFY BELOW THIS LINE.` line.  Rules added by the user above this line will be retained.
 
+Gopkg.toml's without the `# DO NOT MODIFY BELOW THIS LINE.` will be ignored.
+
 {% sample lang="bash" %}
 ```bash
-kubebuilder update vendor
+kubebuilder update
```

Comment:
hmm.... thinking if we should keep "vendor" arg there. Its very explicit which subcomponent are you updating and is generally a good thing.

Reply:
done